bin/run_ascat.r

```diff
@@ -52,7 +52,7 @@ if(opt$gender=="XY"){
 
 
 #GC wave correction
-ascat.bc = ascat.GCcorrect(ascat.bc, opt$gcfile)
+#ascat.bc = ascat.GCcorrect(ascat.bc, opt$gcfile)
```

Comment:
Are you sure about that one?

Reply:
```suggestion
ascat.bc = ascat.GCcorrect(ascat.bc, opt$gcfile)
```

Reply:
fixing the `conf/igenomes.config` should do the trick ;-)

Reply:
No, the best would be to solve the gcfile issue, I had an error there.

Reply:
Yes, I believe you had that issue because the file was actually not existing, on my server, this file was `input.5` instead of the actual file...